src/planscape/Dockerfile

```diff
@@ -1,8 +1,28 @@
-FROM python:3
+FROM rocker/tidyverse as final
```

Comment:
This seems to pull in a lot including RStudio (https://hub.docker.com/r/rocker/tidyverse). Is it necessary?

Reply:
tidyverse will be necessary (and can take some time to install), but RStudio isn't necessary.

I looked into multistage builds briefly, but there isn't much documentation for copying tidyverse packages from a build image to a final image. I can try again.

image-size-wise, Django + R (installed through apt) was 1.83GB, while Django + rocker/tidyverse (obtained from a base image) + forsys was 2.78GB, which suggests there's some room for improvement.